### ISSUE TITLE
Duration shown in the instructions might be far too short for short ferries.

### ISSUE THREAD

**Issue by ratrun** [graphhopper/graphhopper#523]
This occurs for very short ferries, which are tagged with a duration tag which results in a speed slower to the lowest speed, which can be encoded. One example is here, where the duration tag is set to 10 minutes:

https://graphhopper.com/maps/?point=52.019398%2C9.423244&point=52.019015%2C9.426355&locale=de-AT&vehicle=car&weighting=fastest&elevation=false&layer=Omniscale

This issue probably should be solved toghether with #455, which should indicate that also no waiting time is included within the calculated route duration.

**Comment by karussell** [comment]
Yeah, this is important and could be solved with a more dynamic encoded speed value (smaller differences possible for smaller values). Or with an additional time offset (delay) encoded in the flags, which is also useful at other places. We'll see once the flexibility stuff will get more shape.

> This issue probably should be solved toghether with #455, which should indicate that also no 
> waiting time is included within the calculated route duration.

Although the duration tag of the ferry route does not include waiting time, I would roughly estimate and add such to the instructions. E.g. adding `duration/2` to the duration and at least 10min or something.

**Comment by spixi** [comment]
See also #3156 
https://graphhopper.com/maps/?point=51.411908%2C6.323646&point=51.411983%2C6.324901&profile=foot&layer=Omniscale

That issue is slightly different, because it is a manual cable ferry, which is only on the correct side of the river half the time. That means, a duration of 00:03 should be counted as 4.5 minutes in average.

**Comment by spixi** [comment]
> > because it is a manual cable ferry, which is only on the correct side of the river half the time.
> 
> Do you think this assumption is correct for most ferrys tagged as `ferry:cable=yes`?

It depends. There are ferries operating on a specific schedule (those may be indicated using `interval`) and ferries operating on demand. There is also a difference between a commissioned ferry and a self-service ferry. Short cable ferries usually only have one vessel, which can only be on one side of the river.
